client/state/stats/lists/test/utils.js

```diff
@@ -1114,5 +1114,59 @@ describe( 'utils', () => {
 				} );
 			} );
 		} );
+
+		describe( 'statsVisits()', () => {
+			it( 'should return an empty array if not data is passed', () => {
+				const parsedData = normalizers.statsVisits();
+				expect( parsedData ).to.eql( [] );
+			} );
+
+			it( 'should return an empty array if the payload no data attribute', () => {
+				const parsedData = normalizers.statsVisits( { bad: [] } );
+
+				expect( parsedData ).to.eql( [] );
+			} );
+
+			it( 'should return an a properly parsed data array', () => {
+				const parsedData = normalizers.statsVisits( {
+					fields: [ 'period', 'views', 'visitors' ],
+					data: [
+						[ '2016-12-22', 0, 0 ],
+						[ '2016-12-23', 10, 6 ]
```

Comment:
Not necessary but if you used a date that falled on a weekend here, then you could test that `className` logic too.